knowledge-base/gauge-pointer-tooltip.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tooltip for the Gauge Pointer
```

Comment:
```suggestion
title: Show a Tooltip for the Gauge Pointer
```

Reply:
Articles can be omitted in titles ("Show Tooltip for the Gauge Pointer")

Reply:
@yordan-mitev , @dimodi , can we please sync on this and the below point, so we can move forward with merging the pr?

Reply:
> @yordan-mitev , @dimodi , can we please sync on this and the below point, so we can move forward with merging the pr?

Dimo is right, omitting the article is allowed in titles as far as the meaning is still comprehensible. It is up to you to decide.